### DISCUSSION TITLE
Welcome to unstoppable-wallet-android Discussions!

### DISCUSSION THREAD

**Discussion by horsys** [horizontalsystems/unstoppable-wallet-android#5338] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 3point14159** [comment]
Why post anything here when nobody from the community responds or engages? Is there a better venue where discussion is actually discussed? Because ain't nothing happening around here.

**Reply by shadyshoo** [reply]
I'm back sorry

**Comment by shadyshoo** [comment]
Hi hold on its unstoppable
